email_account_validity/_base.py

```diff
@@ -40,9 +45,11 @@ def __init__(
         self._store = store
 
         self._period = config.period
+        self._send_links = config.send_links
 
         (self._template_html, self._template_text,) = api.read_templates(
             ["notice_expiry.html", "notice_expiry.txt"],
+            os.path.join(os.path.dirname(os.path.abspath(__file__)), "templates"),
         )
```

Comment:
Note to self: revisit once https://github.com/matrix-org/synapse/pull/10587 and https://github.com/matrix-org/synapse/pull/10596 land

Reply:
Both PRs have been merged, is there anything more needed here?